src/mgos_onewire.c

```diff
@@ -34,7 +34,9 @@ struct mgos_onewire_search_state {
 };
 
 struct mgos_onewire {
-  int pin;
+  bool separate_io;
```

Comment:
isn't it the same as `pin_in != pin_out`?

Reply:
it is